wikimedia/wikipedia-ios
# [ALT TEXT] Sprint 1 logging

**Phabricator:** https://phabricator.wikimedia.org/T371428

### Notes
This PR adds logging for the Sprint 1 tasks, as described in https://phabricator.wikimedia.org/T371428#10042958.

Note I am only logging the group assignment action data for the `group_assignment` action, not the other events, per our discussion last week. I will confirm this is still correct since the deck doesn't reflect this change.

### Test Steps
(Optional since I have already gone through these, we can lean on QA for this).
1. Show all console logging in WMFLogging.h
2. Fresh install, enable all toggles in Developer Settings, and go through image recommendations flow.
3. You should see group assignment log whether or not the alt text prompt appears.
4. When alt text prompt appears, confirm impression is logged and tapping buttons log actions.
5. When half sheet modal appears, confirm impression is logged. Confirm focusing textview is logged and minimizing half sheet is logged.
6. Confirm submitting alt text logs success action with action data.
